lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb

```diff
@@ -601,7 +605,7 @@ def process_assertion(f, message, args)
         rescue SyntaxError => err
           $miq_ae_logger.error("Assertion had the following Syntax Error: '#{err.message}'")
           raise MiqAeException::AssertionFailure, "Syntax Error in Assertion: <#{assertion}>"
-        rescue Exception => err
```

Comment:
We are running ruby scripts here with the `eval` and do need to rescue `Exception`.  There are a few others we need to keep as well.

Reply:
That's right! Will update it